js/ui/drop_down_button.js

```diff
@@ -588,7 +598,7 @@ const DropDownButton = Widget.inherit({
                 this._createSelectionChangedAction();
                 break;
             case 'deferRendering':
-                if(!value && !this._popup) {
+                if(windowUtils.hasWindow() && !value && !this._popup) {
```

Comment:
See this comment:
https://github.com/DevExpress/DevExtreme/pull/11847/files#diff-cc46f04faa146e60076fc47eff5a47c1R427

Reply:
fixed